bdemeta/__main__.py

```diff
@@ -74,6 +74,21 @@ def get_parser() -> argparse.ArgumentParser:
 
     return parser
 
+def normalize_roots(roots: List[pathlib.Path], config_dir: pathlib.Path) -> List[pathlib.Path]:
+    result = []
+    for root in roots:
+        path = pathlib.Path(root)
+        if path.is_absolute():
+            result.append(path)
+        else:
+            result.append(config_dir/path)
+
+    for root in result:
+        if not root.is_dir():
+            raise InvalidPathError(root)
+
+    return result
+
```

Comment:
Thanks for pulling this out into its own function. It would be a good idea to move it into `bdemeta/resolver.py` and write a separate unit test for it.

Reply:
Done.